app/views/email_signup_information/show.html.erb

```diff
@@ -1,22 +1,24 @@
 <% page_title @email_signup_information.title %>
-<% page_class "email-signup-information-show" %>
+<% page_class "govuk-width-container" %>
 
-<div class="inner-block">
-  <%= render "govuk_publishing_components/components/title", {
-    context: "Email alert subscription",
-    title: @email_signup_information.title,
-    margin_bottom: 4,
-  } %>
-</div>
-
-<div class="inner-block">
-  <p><%= @email_signup_information.summary %></p>
-  <div class='choices'>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds ">
+    <%= render "govuk_publishing_components/components/title", {
+      context: "Email alert subscription",
+      title: @email_signup_information.title,
+      margin_bottom: 4,
+    } %>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: @email_signup_information.summary ,
+      margin_bottom: 8,
+    } %>
     <% @email_signup_information.email_signup_pages.each do |page| %>
       <%= render "govuk_publishing_components/components/heading", {
-        text: page.text
+        text: page.text,
       } %>
-    <p><%= page.description %></p>
+    <p class="govuk-body govuk-margin-bottom-6">
```

Comment:
```suggestion
    <p class="govuk-body govuk-!-margin-bottom-6">
```

Reply:
Nice spot, realised this wasn't needed so was removed entirely.